wildfly-builder-image/image.yaml

```diff
@@ -3,7 +3,7 @@ schema_version: 1
 name: &imgName "wildfly/wildfly-centos7"
 description: "The WildFly s2i image"
 version: &imgVersion "dev"
-from: "centos/s2i-base-centos7"
```

Comment:
Hi @sudeshnas93 , here you are changing the base image of the builder image. It will break the image build.
What you should do is add a label:
- name: name
  value: *imgName
This should also be applied to the runtime image: https://github.com/wildfly/wildfly-s2i/blob/master/wildfly-runtime-image/image.yaml#L7
Thank-you.